app/src/main/java/org/wikipedia/page/ToCHandler.java

```diff
@@ -152,6 +152,7 @@ void log() {
     }
 
     void scrollToSection(@NonNull String sectionAnchor) {
+        bridge.execute(JavaScriptActionHandler.prepareToScrollTo(sectionAnchor, "{ highlight: false }"));
         for (Section section : adapter.sections) {
             if (section.getAnchor().equals(sectionAnchor)) {
                 scrollToSection(section);
```

Comment:
Do we still need this other `scrollToSection()` call, then? It sounds like we're making it scroll twice.

Reply:
same

Reply:
To clarify: the "scroll_to_anchor" handler will automatically scroll to the desired section anyway. So why do we need to do `scrollToSection()` here a second time?